plugins/modules/iosxr_ospfv2.py

```diff
@@ -0,0 +1,2546 @@
+      - process_id: 10
+        authentication:
+          keychain: ansible_test1102
+        areas:
+        - area_id: 11
+          default_cost: 5
+          cost: 11
+        - area_id: 22
+          default_cost: 6
+      - process_id: 30
+        areas:
+        - area_id: 11
+          default_cost: 5
+        - area_id: 22
+          default_cost: 6
+
+        cost: 2
+        default_metric: 10
+        transmit_delay: 2
+        hello_interval: 1
+        dead_interval: 2
+        retransmit_interval: 2
+        weight: 2
+        packet_size: 577
+        priority: 1
+        router_id: 2.2.2.2
+        demand_circuit: enable
+        passive: disable
+        summary_in: enable
+        flood_reduction: disable
+        mtu_ignore: enable
+        external_out: disable
+    state: rendered
+
+#
+#
+# -------------------------
+# Module Execution Result
+# -------------------------
+#
+#
+# "rendered": [
+#        "router ospf 27",
+#        "area 10 authentication keychain ansi11393",
+#        "area 10 hello-interval 2",
+#        "router ospf 26",
+#        "adjacency stagger 10 20",
+#        "authentication message-digest keychain ansible1101pass",
+#        "router ospf 10",
+#        "authentication keychain ansible_test1102",
+#        "area 11 default-cost 5",
+#        "area 11 cost 11",
+#        "area 22 default-cost 6",
+#        "router ospf 30",
+#        "cost 2",
+#        "weight 2",
+#        "passive disable",
+#        "priority 1",
+#        "flood-reduction disable",
+#        "default-metric 10",
+#        "router-id 2.2.2.2",
+#        "demand-circuit enable",
+#        "packet-size 577",
+#        "transmit-delay 2",
+#        "summary-in enable",
+#        "external-out disable",
+#        "dead-interval 2",
+#        "hello-interval 1",
+#        "retransmit-interval 2",
+#        "mtu-ignore enable",
+#        "area 11 default-cost 5",
+#        "area 22 default-cost 6"
+#    ]
+
+
+# Using gathered
+#
+# Before state:
+# -------------
+#
+# RP/0/RP0/CPU0:anton#show running-config router ospf
+# Thu Jun 11 16:06:44.406 UTC
+# router ospf 10
+#  authentication keychain ansible_test1102
+#  area 11
+#   cost 11
+#   default-cost 5
+#  !
+#  area 22
+#   default-cost 6
+#  !
+# !
+# router ospf 26
+#  authentication message-digest keychain ansible1101pass
+#  adjacency stagger 10 20
+# !
+# router ospf 27
+#  area 10
+#  authentication keychain ansi11393
+#   hello-interval 2
+# !
+# !
+# router ospf 30
+#  router-id 2.2.2.2
+#  summary-in enable
+#  external-out disable
+#  cost 2
+#  packet-size 577
+#  weight 2
+#  passive disable
+#  priority 1
+#  mtu-ignore enable
+#  flood-reduction disable
+#  dead-interval 2
+#  retransmit-interval 2
+#  demand-circuit enable
+#  hello-interval 1
+#  transmit-delay 2
+#  default-metric 10
+#  area 11
+#   default-cost 5
+#  !
+#  area 22
+#   default-cost 6
+#  !
+# !
+#
+- name: Gather ospfv2 routes configuration
+  cisco.iosxr.iosxr_ospfv2:
+    config:
```

Comment:
This may work, but probably not a good idea to keep this as an example for `gathered`
```suggestion
```

Reply:
yeah, this works but as it's not necessary I'll remove this. Thanks